### PR TITLE
add airdrop table

### PR DESCRIPTION
**Pull request by groob**
Query Airdrop discovery for the primary user.

```
osquery> select * from kolide_airdrop_preferences;
+----------+-------------+
| username | discover_by |
+----------+-------------+
| victor   | Everyone    |
+----------+-------------+

```

### CODE REVIEW

**Comment by terracatta** [comment]
Once merged would love to ship to customers ASAP so we can fix the bad practice.

File: osquery/table/corefoundation_helpers_darwin.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+*/
+import "C"
+import (
+	"fmt"
+	"reflect"
+	"unsafe"
+)
+
+func copyValue(key, domain, username string) C.CFPropertyListRef {
+	return C.CFPreferencesCopyValue(
```
Comment (zwass) [inline osquery/table/corefoundation_helpers_darwin.go]: Do we need an associated `CFRelease` with this? https://developer.apple.com/library/archive/documentation/CoreFoundation/Conceptual/CFMemoryMgmt/Concepts/Ownership.html#//apple_ref/doc/uid/20001148-103029

File: osquery/table/corefoundation_helpers_darwin.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+		CFStringRef(key),
+		CFStringRef(domain),
+		CFStringRef(username),
+		C.kCFPreferencesAnyHost,
+	)
+}
+
+func CFStringRef(s string) C.CFStringRef {
+	return C.CFStringCreateWithCString(C.kCFAllocatorDefault, C.CString(s), C.kCFStringEncodingUTF8)
```
Comment (zwass) [inline osquery/table/corefoundation_helpers_darwin.go]: I think by the same "Create Rule" we need to `CFRelease` this string.

File: osquery/table/corefoundation_helpers_darwin.go [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+	val := C.CFPreferencesCopyValue(
+		keyCFString, domainCFString, usernameCFString, C.kCFPreferencesAnyHost,
+	)
+	defer C.CFRelease((C.CFTypeRef)(val))
+	return goValueFromCFPlistRef(val)
+}
+
+// CFStringRef returns a C.CFStringRef which must be released with C.CFRelease
+func CFStringRef(s string) C.CFStringRef {
```
Comment (zwass) [inline osquery/table/corefoundation_helpers_darwin.go]: should this and the below functions be private?

Comment (groob) [reply osquery/table/corefoundation_helpers_darwin.go]: Yeah, no reason for them to be public, I copied them from my own gist i found somewhere.

**Pull request merged**